vscode-html/compile-vscode-html-languageserver.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# @see https://github.com/mattn/vim-lsp-settings/pull/48
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="${SCRIPT_DIR}"
+SRC_DIR="${REPO_DIR}/vscode-html-languageserver"
+DIST_DIR="${REPO_DIR}/out"
+
+
+# ------------------------- #
+# download the source codes #
+# ------------------------- #
+
+pushd "${REPO_DIR}" || exit
+
+rm -rf "${SRC_DIR}" "${DIST_DIR}" "package.json" "package-lock.json"
+
+# or get the source via git clone
+# git clone --depth=1 https://github.com/vscode-langservers/vscode-html-languageserver "${SRC_DIR}"
+
+wget https://github.com/vscode-langservers/vscode-html-languageserver/archive/master.zip -O src.zip
+unzip src.zip
+rm -f src.zip
+mv vscode-html-languageserver-master "${SRC_DIR}"
+
+popd || exit
+
+
+# ------------ #
+# prepare deps #
+# ------------ #
+
+pushd "${SRC_DIR}" || exit
+
+npm install
+npm install -D typescript
```

Comment:
I'm sorry but I was wrong.
It seems like at least one file requires it so it's needed in production too.
Saw it in `vscode-html/out/modes/javascriptSemanticTokens.js`

Reply:
No -D in this one :)

Reply:
Should be no -D in this one :)

Reply:
nice catch